consensus/src/creation/mod.rs

```diff
@@ -107,7 +107,11 @@ pub async fn run<H: Hasher>(
     } = io;
     let starting_round = futures::select! {
         maybe_round =  starting_round => match maybe_round {
-            Ok(round) => round,
+            Ok(Some(round)) => round,
+            Ok(None) => {
+                warn!(target: "AlephBFT-creator", "None starting round provided. Exiting.");
```

Comment:
In this case `Creator` ends, so the `Runway` will terminate. Because of that `Member` will terminate. So in aleph-node authority task will terminate. In this case aleph-node will not crash, but during this round it will not be an authority. I think this is a correct approach. If we get None it means, that our backup for some reason is broken and we might be byzantine without knowing about it.